plugins/modules/cpanm.py

```diff
@@ -56,6 +56,13 @@
       - Only install dependencies.
     type: bool
     default: false
+  withrecommands:
+    description:
+      - Installs dependencies declared as recommends per META spec.
+      - When these dependencies fail to install, cpanm continues the installation, since they are just recommendation.
+    type: bool
+    default: false
```

Comment:
I will suggest we leave it without default, because then we use that parameter to also implement the counterpart param `--without-recommend`
```suggestion
```

Reply:
Agreed!